stock_move_location/README.rst

```diff
@@ -92,12 +92,15 @@ Contributors
 
 * Mathieu Vatel <mathieu@julius.fr>
 * Mykhailo Panarin <m.panarin@mobilunity.com>
-* Sergio Teruel <sergio.teruel@tecnativa.com>
 * Joan Sisquella <joan.sisquella@forgeflow.com>
 * Jordi Ballester Alomar <jordi.ballester@forgeflow.com>
 * Lois Rilo <lois.rilo@forgeflow.com>
 * Héctor Villarreal <hector.villarreal@forgeflow.com>
 
+* Tecnativa <tecnativa.com>
+  * Sergio Teruel
```

Comment:
This hasn't been added correctly for the syntax: the empty line must be between these lines, not above

Reply:
Oh sorry, MB. Already fixed.